Source/API/EbSvtAv1Enc.h

```diff
@@ -63,6 +63,24 @@ typedef enum {
     SUPERRES_MODES
 } SUPERRES_MODE;
 
+typedef enum {
+    SVT_AV1_STREAM_INFO_START = 1,
+
+    // SvtAv1FixedBuf
+    SVT_AV1_STREAM_INFO_FIRST_PASS_STATS_OUT = SVT_AV1_STREAM_INFO_START,
+
+    SVT_AV1_STREAM_INFO_END,
+} SVT_AV1_STREAM_INFO_ID;
+
+/*!\brief Generic fixed size buffer structure
+ *
+ * This structure is able to hold a reference to any fixed size buffer.
+ */
+typedef struct SvtAv1FixedBuf {
+    void *buf;       /**< Pointer to the data. Does NOT own the data! */
+    uint64_t sz;       /**< Length of the buffer, in chars */
```

Comment:
```suggestion
    size_t sz;       /**< Length of the buffer, in chars */
```

maybe better to use `size_t` since it's usually used to represent objects etc size, even though it might be the same size as uint64_t, but it will also match aom's

Reply:
will this conflict with current API, if we search the EbSvtAv1Enc.h, we have no size_t things.